Display event

dipscoder opened the pull request:
Display:
- User's Upcoming Events
- New Events
And
- Made route for individual Event dynamic

@Abhishek-kumar09

dipscoder committed:
Display user's upcoming events

dipscoder committed:
Display Individual event

dipscoder committed:
Display new events

Abhishek-kumar09 commented:
The card is not having perfect margins:
![image](https://user-images.githubusercontent.com/48255244/121682641-f4e08180-cad9-11eb-9552-78f466245875.png)

If there are no bookmarked events then remove that. If search functionality is not woking yet, comment the code for that.

Abhishek-kumar09 commented:
Link checkout buttons for the events to checkout to that event.

dipscoder commented:
> Link checkout buttons for the events to checkout to that event.

Its already completed

SHITIZ-AGGARWAL committed:
Added designs

SHITIZ-AGGARWAL committed:
Add README

Abhishek-kumar09 committed:
Merge pull request #86 from SHITIZ-AGGARWAL/design

Added designs

symaticvisuals committed:
landing page v-1 (#59)

* landing page v-1

* landing page v-1

* landing page v-1

* page v2

* v2

* Bump typescript from 3.9.9 to 4.3.2

Bumps [typescript](https://github.com/Microsoft/TypeScript) from 3.9.9 to 4.3.2.
- [Release notes](https://github.com/Microsoft/TypeScript/releases)
- [Commits](https://github.com/Microsoft/TypeScript/compare/v3.9.9...v4.3.2)

---
updated-dependencies:
- dependency-name: typescript
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

* landing page done

* landing page done - generalized fonts

* landing page done - production build

* Create events (#79)

* Initial commit

* Complete eventForm with image upload

* Store images into specific user

* Make cancel btn working

* Format file

* Update

* Added designs

* Add README

* Fix homepage responsiveness

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Abhishek Kumar <abhimait1909@gmail.com>
Co-authored-by: Dipesh Jaiswal <68822438+dipscoder@users.noreply.github.com>
Co-authored-by: SHITIZ-AGGARWAL <shitizaggarwal2000@gmail.com>
Co-authored-by: SHITIZ AGGARWAL <53532851+SHITIZ-AGGARWAL@users.noreply.github.com>

dipscoder committed:
Hide bookmark and searchbar

dipscoder committed:
Display number of events conducted

dipscoder committed:
Remove warnings

dipscoder commented:
@Abhishek-kumar09  Please have a look

Abhishek-kumar09 commented:
![image](https://user-images.githubusercontent.com/48255244/121817524-9533d300-cc9f-11eb-82d0-59d77553a522.png)

Proper padding is necessary.
The cards are having improper padding.
It should not have fixed width or if so try to get a consistent design.

dependabot[bot] committed:
Bump react-helmet from 5.2.1 to 6.1.0 (#93)

Bumps [react-helmet](https://github.com/nfl/react-helmet) from 5.2.1 to 6.1.0.
- [Release notes](https://github.com/nfl/react-helmet/releases)
- [Changelog](https://github.com/nfl/react-helmet/blob/master/CHANGELOG.md)
- [Commits](https://github.com/nfl/react-helmet/commits/6.1.0)

---
updated-dependencies:
- dependency-name: react-helmet
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dipscoder committed:
Fix card styling

Abhishek-kumar09 committed:
Merge commit 'refs/pull/84/head' of https://github.com/codeforcauseorg/causefolio into HEAD

Abhishek-kumar09 commented:
![image](https://user-images.githubusercontent.com/48255244/121967739-8710b000-cd8e-11eb-9b7e-1810377c508e.png)
The design is breaking

Abhishek-kumar09 committed:
fix linting errors and package fixes

dipscoder commented:
> ![image](https://user-images.githubusercontent.com/48255244/121967739-8710b000-cd8e-11eb-9b7e-1810377c508e.png)
> The design is breaking

Everything is working fine in my local

![Screenshot (355)](https://user-images.githubusercontent.com/68822438/122002573-8b10f200-cdcf-11eb-9c27-63c37691a8f7.png)

Abhishek-kumar09 commented on src/components/UserNewEvents.js:
```diff
@@ -131,10 +131,22 @@ function UserNewEvents() {
+                style={event?.eventName.length <= 11 ? { marginTop: '21px', backgroundColor: 'white' } : { marginTop: '1px', backgroundColor: 'white' }}
                 className={classes.checkOut}
-                onClick={() => history.push(`/events/${eventID[idx]}`)}
+                // onClick={() => history.push(`/events/${eventID[idx]}`)}
```
isko revert kar dena

dipscoder replied:
Ok

Abhishek-kumar09 reviewed (commented):
The code was not working  for the non logged in users. 
Have you tried it?
It was showing null error on user which was faced by many other contributors.

I did some changes in package file and I added it in this PR. Please give a look correct errors if any.

dipscoder commented:
> 
> 
> The code was not working for the non logged in users.
> Have you tried it?
> It was showing null error on user which was faced by many other contributors.
> 
> I did some changes in package file and I added it in this PR. Please give a look correct errors if any.

Just because right now, Profile can only be viewed by logged in users
Current route for profile: "/profile"

I will make this dynamic so that anyone can view any other user's profile 
Updated route will be: " /profile/firstname-lastname-4digitsOfUserID"

Is it Ok if I use this route? 
Also, should I create a new PR for this or add changes in current PR only?

dipscoder committed:
Fix New Event checkout btn

dipscoder committed:
Fix user and eventID null problem

dipscoder committed:
Fix

dipscoder committed:
formatting

dipscoder committed:
Add parentheses

dipscoder committed:
Remove log statements and other things

dipscoder committed:
Remove spaces

dipscoder commented:
@Abhishek-kumar09  please have a look.

Abhishek-kumar09 commented:
Everything is working fine right, with or without logIn?

Abhishek-kumar09 commented:
Actually, I have not checked it locally, so confirming your side.

dipscoder commented:
Yes, everything is working fine
